src/utils/safe-messages.ts

```diff
@@ -29,6 +29,14 @@ const EIP1271_SUPPORTED_SAFE_VERSION = '1.0.0'
 
 const EIP1271_OFFCHAIN_SUPPORTED_SAFE_APPS_SDK_VERSION = '7.11.0'
 
+/**
+ * Typeguard for EIP712TypedData
+ *
+ */
+export const isObjectEIP712TypedData = (obj: any): obj is EIP712TypedData => {
```

Comment:
```suggestion
export const isEIP712TypedData = (obj: any): obj is EIP712TypedData => {
```

IMHO a better name + automatic imports in the IDE won't confuse it with the apps SDK's one.